app/src/components/Map/LayerBlendingOptionsTooltip.jsx

```diff
@@ -46,6 +47,23 @@ class LayerOptionsTooltip extends Component {
       opacityRangeValue: this.opacityRangeConfig.value,
       hueRangeValue: this.hueRangeConfig.value
     };
+
+    this.closeTooltip = this.closeTooltip.bind(this);
+  }
+
+  shouldComponentUpdate(nextProps) {
```

Comment:
`shouldComponentUpdate` is not the right place for that. If your `shouldComponentUpdate` implementation always returns the same value, it means you should not use `shouldComponentUpdate`.
Use another React comp lifecycle hook, like `componentWillUpdate()`, `componentDidUpdate()` or even `render` depending on the use case.